docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+version: '3.8'
+
+services:
+  mini-tree:
+    image: sifnoc/summa-aggregation:mini-tree-v0.1
```

Comment:
Is this a correct path even for other users?

Reply:
Yes, anyone can access it. You can pull image with  this command `docker pull sifnoc/summa-aggregation:mini-tree-v0.1`.

Eventually, we plan to upload the image to the summa-dev account (which I believe has not been created yet). Once that's done, the final image name is expected to be summa-dev/summa-aggregation:worker.